packages/frontend/src/components/wallet/NFTBox.js

```diff
@@ -130,18 +157,23 @@ const NFTBox = ({ tokenDetails }) => {
             {
                 ownedTokensMetadata &&
                 <div className='tokens'>
-                    {ownedTokensMetadata.map(({ token_id, metadata: { mediaUrl, title } }) => {
+                    {ownedTokensMetadata.map(({ token_id, metadata }) => {
+                        const { mediaUrl, title } = metadata;
+
                         return <div className='nft' key={token_id}>
```

Comment:
Can we please add the `onClick` to the `div` instead of two separate ones for the `img` and title `div`. 

Also, let's use the `redirectTo` action instead of `history.push` :)

Reply:
Can we also wrap the return result in a parenthesis so that's it's easier to see what's included:)